src/languages/gams.js

```diff
@@ -0,0 +1,30 @@
+module.exports = function (hljs){
```

Comment:
This won't let the build system to properly wrap this file. A language file should contain a bare JavaScript function and some meta data (http://highlightjs.readthedocs.org/en/latest/language-contribution.html#provide-meta-data).